frontend_tests/node_tests/list_render.js

```diff
@@ -10,6 +10,7 @@ function Element() {
     return { };
 }
 set_global('Element', Element);
+set_global('ui', {});
```

Comment:
@sahil839 I cherry-picked our combined effort at fixing this.

Reply:
Okay, then I will make separate PR for fixing other lists.